Nimiq.d.ts

```diff
@@ -41,7 +41,35 @@ declare namespace Nimiq {
         static toHex(buf: Uint8Array): string
     }
 
-    class SerialBuffer extends Uint8Array {}
+    class SerialBuffer extends Uint8Array {
+        constructor(bufferOrArrayOrLength: any)
+        subarray(start: number, end: number): Uint8Array
+        readPos: number
+        writePos: number
+        reset(): void
+        read(length: number): Uint8Array
+        write(array: any): void
```

Comment:
any => SerialBuffer?

Reply:
I took the annotations directly from the core, and they are using `{*}`. So we have to honor that here.